File: csharp/roman-numerals/RomanNumerals.cs
using System;
using System.Linq;

public static class RomanNumeralExtension
{
    public static string ToRoman(this int value) =>
        string.Join("", Enumerable.Repeat("I", value))
            .Replace("IIIII", "V")
            .Replace("IIII", "IV")
            .Replace("VV", "X")
            .Replace("VIV", "IX")
            .Replace("XXXXX", "L")
            .Replace("XXXX", "XL")
            .Replace("LL", "C")
            .Replace("LXL", "XC")
            .Replace("CCCCC", "D")
            .Replace("CCCC", "CD")
            .Replace("DD", "M")
            .Replace("DCD", "CM")
        ;
}

File: kotlin/roman-numerals/src/main/kotlin/RomanNumerals.kt
object RomanNumerals {

    fun value(n: Int) = "I".repeat(n)
        .replace("IIIII", "V")
        .replace("IIII", "IV")
        .replace("VV", "X")
        .replace("VV", "X")
        .replace("VIV", "IX")
        .replace("XXXXX", "L")
        .replace("XXXX", "XL")
        .replace("LL", "C")
        .replace("LXL", "XC")
        .replace("CCCCC", "D")
        .replace("CCCC", "CD")
        .replace("DD", "M")
        .replace("DCD", "CM")
}
